packages/types/src/documents/app/automation/StepInputsOutputs.ts

```diff
@@ -219,7 +219,18 @@ export type GenerateTextStepOutputs = {
   success: boolean
   response?: string
 }
+export type ExtractFileDataStepInputs = {
+  file: RowAttachment | string
+  source: "URL" | "Attachment"
+  fileType?: string
+  schema: Record<string, any>
+}
 
+export type ExtractFileDataStepOutputs = {
+  response?: string
+  success: boolean
+  data?: Record<string, any>
```

Comment:
Should we use smarter types such as `{ success: false } || { success: true, data: Record<string, any>}`, making data required for successful calls?